### PR TITLE
Add collected documentation and license badge

### PR DESCRIPTION
**Pull request by Swaagie**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -4,16 +4,22 @@ We take security very seriously at GoDaddy. We appreciate your efforts to
 responsibly disclose your findings, and will make every effort to acknowledge
 your contributions.
 
-## Where should I report security issues?
+## Disclosure policy
```
Comment (3rd-Eden) [inline SECURITY.md]: I would highly advice keeping this file insync with the version provided the our OSWG.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+## Attribution of Changes
+
+When contributors submit a change to this project, after that change is
+approved, other developers with commit access may commit it for the author.
+When doing so, it is important to retain correct attribution of the
+contribution. Generally speaking, Git handles attribution automatically.
+
+## Code Documentation
+
+Ensure that every function in `warehouse.ai` is documented and follows the
```
Comment (msluther) [inline CONTRIBUTING.md]: >  `warehouse.ai`
Should that be `extract-config`?

File: SECURITY.md (hunk 1 above)
Comment (Swaagie) [reply SECURITY.md]: Not feeling strong about it, but I what about specific security information like https://github.com/godaddy/warehouse.ai/blame/master/SECURITY.md#L23. Enforcing all files to be similar just for the sake of it feels restricting.

File: CONTRIBUTING.md (hunk 2 above)
Comment (Swaagie) [reply CONTRIBUTING.md]: Yes, will change it so that this becomes repo agnostic, e.g. `this module` or `this project`

File: SECURITY.md (hunk 1 above)
Comment (Swaagie) [reply SECURITY.md]: This is now in sync

**Pull request merged**